sdk/eventhubs/azure-messaging-eventhubs/src/partition_client_models.cpp

```diff
@@ -14,25 +14,25 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Models {
     os << "StartPosition:[";
     if (sp.Offset.HasValue())
     {
-      os << "Off: " << sp.Offset.Value() << std::endl;
+      os << "Off: " << sp.Offset.Value();
```

Comment:
we dropped the newline, but do we need some comma instead as a separator so that the value and the next text don't come glued together?

Reply:
`Log::Stream` inserts a newline at the end of every line. The `std::endl` was redundant, so I removed it.